python/lsst/daf/butler/remote_butler/server/_server.py

```diff
@@ -54,7 +53,6 @@ def create_app() -> FastAPI:
     config = load_config()
 
     app = FastAPI()
-    app.add_middleware(GZipMiddleware, minimum_size=1000)
```

Comment:
This was preventing the keep-alives from working because it would buffer/compress them and never send them.

I tested it and it turns out that gzipping was slowing down queries by 10% on the RSP -- there's oodles of bandwidth there and apparently the compression overhead is non-trivial.  So I just removed this, since there are  no configuration options to make it only apply to certain content-types or routes.